chef/cookbooks/provisioner/recipes/shell-vars.rb

```diff
@@ -0,0 +1,41 @@
+# Copyright 2015, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+env = node[:provisioner][:config][:environment]
+
+node_vars = []
+role_vars = []
+
+search(:node, "provisioner_config_environment:#{env}") do |n|
+  aliaz = n["crowbar"]["display"]["alias"] rescue nil
+  next unless aliaz && !aliaz.empty?
+  short = aliaz.gsub("node", "n")
```

Comment:
Not sure about this `gsub`; this is a bit too magic for people. If they really wanted to use a shorter version, then they would surely have used `n` directly?